data-source/src/test/scala/tech/ytsaurus/spyt/format/TestTableSettings.scala

```diff
@@ -2,17 +2,16 @@ package tech.ytsaurus.spyt.format
 
 import org.apache.spark.sql.types.StructType
 import tech.ytsaurus.spyt.serializers.SchemaConverter.{SortOption, Unordered}
-import tech.ytsaurus.spyt.serializers.SchemaConverter
+import tech.ytsaurus.spyt.serializers.{SchemaConverter, WriteSchemaConverter, YtLogicalType}
 import tech.ytsaurus.spyt.wrapper.table.YtTableSettings
-import tech.ytsaurus.spyt.serializers.{SchemaConverter, YtLogicalType}
 import tech.ytsaurus.ysontree.YTreeNode
 
 case class TestTableSettings(schema: StructType,
                              isDynamic: Boolean = false,
                              sortOption: SortOption = Unordered,
                              writeSchemaHint: Map[String, YtLogicalType] = Map.empty,
                              otherOptions: Map[String, String] = Map.empty) extends YtTableSettings {
-  override def ytSchema: YTreeNode = SchemaConverter.ytLogicalSchema(schema, sortOption, writeSchemaHint)
+  override def ytSchema: YTreeNode = WriteSchemaConverter(Map.empty, typeV3Format = false).ytLogicalSchema(schema, sortOption)
```

Comment:
Why you explicitly specify constructor parameters when you have defaults of the same values?